### ISSUE TITLE
Update View cart and Checkout buttons

### ISSUE THREAD

**Issue by Geneus** [jameskoster/woocommerce-cart-tab#45]
When you use a different language that translates the ‘cart’ text string to, say the English UK translation i.e. ‘basket’, the “View cart” button gets borked. 

I expanded the buttons' width to 100% and added a top margin declaration of 0.625em to solves this. See pull request #44. 

Cheers!

**Comment by jameskoster** [comment]
But 'Basket' is shorter than 'View cart'? :p 

I tested 'View Basket' and that fit just fine as well;

![fits](https://cldup.com/t6kx_FyKd8-3000x3000.png)

Would you mind including some more details before I merge?

**Comment by Geneus** [comment]
Hmmm… that's odd.

I'm working locally using a child theme. Haven't started any customisations yet. 

To maintain my sanity, I tried it on a fresh install with dummy data imported. Activated WooCommerce and Cart Tab only and can confirm that you're right; they do display inline, one after the other. 

![cart-tab](https://cloud.githubusercontent.com/assets/4223644/26704235/fd75bda8-472d-11e7-8516-aa0952cdfcb4.png)

I'll troubleshoot what could have happened on the site I'm working on and report back! I suspect a plugin's the cause. I'll close the PR and open a new one if the need arises.

With that being said, I just noticed the “Your Cart” header is not translated. Same in your screenshot.
